### PR TITLE
bed management breadcrumb link fixed

### LINKED ISSUE #2504: Broken link in breadcrumbs for bed management
**Describe the bug**
Manage beds page for a particular location at a particular facility generates breadcrumb with location id in it. On clicking the location id it takes to following URL /facility/:facilityID/location/:locationID which is not mapped to any page.

**To Reproduce**

1. Go to a particular facility page
2. Click on 'Location Management
3. Click on 'Manage Beds' for any location
4. Now click on the location id generated in the breadcrumb which will display an empty page

**Screenshots**
Screenshot from 2022-05-24 23-10-23

### PR DESCRIPTION
**Pull request by ishanExtreme**
### Issue Fixed
fixes #2504 

### Updates

- Made the location id link unclickable
- fixed apply styling to breadcrumbs

### Demo

https://user-images.githubusercontent.com/39593226/170127254-75a69446-d23f-4e96-8617-78ff5e7b4d9d.mp4

### CODE REVIEW

File: src/Components/Common/Breadcrumbs.tsx [hunk 1]
```diff
@@ -113,10 +113,7 @@ export default function Breadcrumbs(props: any) {
               crumb.name && (
                 <li
                   key={crumb.name}
-                  className={
-                    "text-sm font-medium text-gray-500 hover:text-gray-700 cursor-pointer " +
-                    crumb.style
-                  }
+                  className={`text-sm font-medium text-gray-500 hover:text-gray-700 cursor-pointer ${crumb.style}`}
```
Comment (developedBySJ) [inline src/Components/Common/Breadcrumbs.tsx]: Use `clsx`

File: src/Components/Facility/BedManagement.tsx [hunk 2]
```diff
@@ -4,8 +4,8 @@ import { useDispatch } from "react-redux";
 import { Button, CircularProgress } from "@material-ui/core";
 import { statusType, useAbortableEffect } from "../../Common/utils";
 import {
-  listFacilityAssetLocation,
-  updateFacilityAssetLocation,
+  // listFacilityAssetLocation,
```
Comment (developedBySJ) [inline src/Components/Facility/BedManagement.tsx]: Cleanup this code

**Comment by nihal467** [comment]
@ishanExtreme @gigincg check the requested change

**Comment by ishanExtreme** [comment]
@nihal467 
Above Issues fixed

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**